docs/source/02_Terminology.md

```diff
@@ -131,7 +132,35 @@ Since in practical terms the time is measured in discrete samples,
 the temporal scope includes the start time sample 
 but does not include the end time sample.
 
-## Time-block [*]
+### Time-block [*]
 
 A contiguous portion of the data recording during which some aspect of the experiment 
 is fixed or noted.
+
+## 2.2 Character sets and restrictions
+
+Starting with HED standard schema versions 8.3.0 and above, HED will allow UTF-8 characters in various settings.
+The types of characters referred to in this specification are:
+
+| Name | Description |
+| ---- | ----------- |
+| `ascii` | utf-8 codes 0 to 127 (single byte) |  
+| `non-ascii` | utf-8 codes greater than 128 (multi-byte) |  
+| `printable` | ascii 32 <= code < 127 |
+|`lowercase` | ascii characters a-z |  
+| `uppercase` | ascii characters A-Z |
+| `letters` | `lowercase` and/or `uppercase` |  
+| `text` | `printable` and/or `non-ascii` |  
+| `digits` | 0-9 |  
+| `blank` | ascii code 32 |  
+| `plus` | ascii code 42 |  
+| `hyphen` | ascii code 45 |
+| `period` | ascii code 46 |  
+| `colon` | ascii code 58 |  
+| `caret` | ascii code 94 |
+| `underscore` | ascii code 95 |
+| `newline` | ascii code 10 (linefeed) |
+| `alphanumeric` | `letters` and/or  `digits` |
+| `name` | `alphanumeric`, `blank`, `hyphen`, `period`, `underscore`, `non-ascii` |
```

Comment:
"#" will have to be allowed on node names in schemas(and sometimes other places)

"/" Will have to be allowed in tags(for def tags specifically)

I don't see hardcoding these as a problem, just noting.

Reply:
The forward slash won't be a tag name it will be a separator.  I separated out the # in the SCHEMA_CHARACTER_INVALID.